services/Invoice.js

```diff
@@ -7,10 +7,45 @@ export default class Invoice extends CRUDService {
         this.endpoint = 'invoices';
     }
 
+    // ----- add all extra methods except the CRUD.
+ 	invoices(){
+        return this.$http.get(this.apiURL + this.endpoint + '/'  ).then((invoices) => invoices.data);
+    }
+
+    count(){
+        return this.$http.get(this.apiURL + this.endpoint + '/'  ).then((invoices) => invoices.data);
+    }
+
+    unpaidCount(){
+        return this.$http.get(this.apiURL + this.endpoint + '/'  ).then((invoices) => invoices.data);
+    }
+
+	unpaidSum(){
+        return this.$http.get(this.apiURL + this.endpoint + '/'  ).then((invoices) => invoices.data);
+    }
+
+	invoice(id){
+        return this.$http.get(this.apiURL + this.endpoint + '/' + id).then((invoice) => invoice.data);
+    }
+
+	add(data){
+        return this.$http.post(this.apiURL + this.endpoint + '/' + data).then((invoice) => invoice.data);
+    }
+
+	remove(id){
+        return this.$http.delete(this.apiURL + this.endpoint + '/' + id).then((invoice) => invoice.data);
+    }
+
+	edit(id, data){
+        return this.$http.put(this.apiURL + this.endpoint + '/' + id, data).then((invoice) => invoice.data);
+    }
```

Comment:
`edit`, `add`, `remove`, `invoice(id), and `invoices()` should be removed since they already exist in `CRUDService` that this extends on.